entrypoint.sh

```diff
@@ -6,16 +6,16 @@ export BASELINE=
 export CUSTOM_RULE_PATH=
 
 if [ "$INPUT_KTLINT_VERSION" = "latest" ]; then
-  curl -sSL https://api.github.com/repos/pinterest/ktlint/releases/latest --header "authorization: Bearer ${INPUT_GITHUB_TOKEN}" |
```

Comment:
@wf9a5m75 can you explain when the issue happens? This line was introduced in #33 to solve the same failure you're mentioning. Without a token, a different rate limit is applied and if many requests are performed from the same IP, you will soon reach that limit.

Reply:
Oh I see. In that case, why don't you implement cache for limited time?

This error occurs you for got the backslash before `line-end`, thus the pipe does not work.

Reply:
Backslash is not needed if the line ends with pipe `|`. Is it always failing for you when you use `ScaCap/action-ktlint`?

Reply:
Without backslash, always fail.

Reply:
@wf9a5m75 coming back to this again, can you replicate this in some public repository so we can take a look?